src/Microsoft.ML.ImageAnalytics/ImageLoader.cs

```diff
@@ -112,6 +133,12 @@ private ImageLoadingTransformer(IHost host, ModelLoadContext ctx)
             // int: id of image folder
 
             ImageFolder = ctx.LoadStringOrNull();
+
+            if (ctx.LoadStringOrNull().Equals("False"))
```

Comment:
>LoadStringOrNull [](start = 20, length = 16)

Use the `ReadBoolByte` and `WriteBoolByte` APIs instead.

Reply:
Also, the comment of the binary format needs to be updated (here and in the Save method), and the version of the transformer needs to be bumped, and we need to handle backwards compatibility. 

---
In reply to: [330463101](https://github.com/dotnet/machinelearning/pull/4242#discussion_r330463101) [](ancestors = 330463101)